pkg/controllers/managementuser/rbac/podsecuritypolicy/version.go

```diff
@@ -0,0 +1,31 @@
+package podsecuritypolicy
+
+import (
+	"errors"
+	"fmt"
+	"strconv"
+
+	v3 "github.com/rancher/rancher/pkg/generated/norman/management.cattle.io/v3"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var errVersionIncompatible = errors.New("podsecuritypolicies are not available in Kubernetes v1.25 and above")
+var genericErrorString = "failed to check cluster version compatibility for podsecuritypolicy controllers: %v"
```

Comment:
the name "generic" isn't too telling, maybe something more related to the contents of the error string?